universe/interface.go

```diff
@@ -169,9 +169,9 @@ type BaseBackend interface {
 	// tree, stored at the base key. The metaReveal type is purely
 	// optional, and should be specified if the genesis proof committed to
 	// a non-zero meta hash.
-	RegisterIssuance(ctx context.Context,
-		key BaseKey, leaf *MintingLeaf,
-		metaReveal *proof.MetaReveal) (*IssuanceProof, error)
+	RegisterIssuance(ctx context.Context, key BaseKey, leaf *MintingLeaf,
+		metaReveal *proof.MetaReveal, replace bool) (*IssuanceProof,
```

Comment:
Is the bool actually needed here? Seems like we can just assume that if we get a new valid proof, but with an updated block hash (actually in the chain), then we can go ahead and accept it.

Reply:
Hmm, yes, if we change `InsertUniverseLeaf` into an upsert, this is possible. Makes more sense, updated.